common/lib/client/resource.ts

```diff
@@ -216,11 +226,11 @@ class Resource {
             /* token has expired, so get a new one */
             rest.auth.authorize(null, null, function (err: Error) {
               if (err) {
-                callback(err);
+                callback(ErrorInfo.fromValues(err));
```

Comment:
Looks like `rest.auth.authorize` will only invoke with an `ErrorInfo` (sorry it should be typed better). You should be able to just change the callback arg type to `ErrorInfo` and not have to use this.

Reply:
Switched it to err: ErrorInfo